apps/web/lib/api/links/process-link.ts

```
@@ -534,6 +535,7 @@ export async function processLink<T extends Record<string, any>>({
   // remove polyfill attributes from payload
   delete payload["shortLink"];
   delete payload["qrCode"];
+  delete payload["keyLength"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)